index.js

```diff
@@ -46,10 +46,11 @@ const getHighWaterMark = (streams, objectMode) => {
 const endWhenStreamsDone = async (passThroughStream, streams) => {
 	try {
 		const abortController = new AbortController();
-		setMaxListeners(streams.length + 1, abortController.signal);
+		setMaxListeners((streams.length * 2) + 2, abortController.signal);
```

Comment:
Would be good with a code comment about the magic numbers here.

Reply:
Done with 177ab80cb7e0cb6ae00b8556cd67119dfb86745a :+1: